docs/UserGuide.md

```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
 
 **Example(s):**<br />
-1. `list ingredient`
+1. Command: `list ingredient`
 <br />Expected Output:<br />
-![listIngredient1.png](images/ingredientCommands/listIngredient1.png)
+![listIngredient1.png](images/ingredientCommands/listIngredient.png)
+
+[\[Back to Table of Contents\]](#table-of-contents)
 
-### 4.2.7 View Ingredients
+#### 4.4.7 View Ingredient
 
-View an ingredient in the [Main Window](#36-mainwindow).
+Open an ingredient in the [Main Window](#36-mainwindow).
 
 **Format:**<br />
 `view ingredient <index>`
 
 **Example(s):**<br />
-1. `view ingredient 3`
-   <br />Expected Output:<br />
-   ![viewCommand1.png](images/ingredientCommands/viewCommand1.png)
+1. Command: `view ingredient 3`
+<br />Expected Output:<br />
+![viewCommand1.png](images/ingredientCommands/viewIngredient.png)
 
 **Additional Information:**<br />
 - An index number is required for the Command. Refer to the indexes displayed for each [Card](#33-cards) in the
   [Side Bar](#32-side-bar).
 
-## 4.3 Recipes
-This sections covers commands related to RecipeBook management. Any command primarily interacting with recipes will
+[\[Back to Table of Contents\]](#table-of-contents)
+  
+### 4.5 Recipes
+This sections covers commands related to Recipe Book management. Any command primarily interacting with recipes will
 be here.
 
-### 4.3.1 Add Recipes
+#### 4.5.1 Add Recipe
 
-Add a recipe to the RecipeBook.
+Add a recipe to the Recipe Book.
 
 **Format:**<br />
-`add recipe -n <name> -i <ingredient> <quantity> [-d <optional description>] [-s <steps>]...`
+`add recipe -n <name> -i <ingredient>... [-d <description>] [-s <steps>]...`
 
 **Example(s):**<br />
-1. `add recipe -n pasta -i tomato 1 -i milk 100ml -i chicken breast 200g -s Chicken thicc -s Thicc chicken`
-    <br />Expected Output:<br />
-    ![addRecipe1.png](images/recipeCommands/addRecipe1.png)
-
-2. `add recipe -n aglio olio -i pasta 200g -d grandmother aglio olio recipe -s aglioli olioli -s aglioli olioli`
-    <br />Expected Output:<br />
-    ![addRecipe2.png](images/recipeCommands/addRecipe2.png)
+1. Minimal command flags used:<br /> 
+Command: `add recipe -n pasta -i tomato 2 -i linguine 50g`
+<br />Expected Output:<br />
+![addRecipe1.png](images/recipeCommands/addRecipe1.png)
 
-3. `add recipe -n Grilled Lamb Chop with Mint Puree -i Mint 5g -i Lamb Chops 1kg -i Butter 20g -i Garlic 20g -s Season
+2. All command flags used:<br />
+Command:`add recipe -n Grilled Lamb Chop with Mint Puree -i Mint 5g -i Lamb Chops 1kg -i Butter 20g -i Garlic 20g -s Season
 the lamb chops with salt and pepper. -s Grill the lamb chops over medium high heat until cooked with butter.
 -s Blend the mint with garlic to make a puree. -s Leave the lamb chops to rest for 5min. -s Serve with mint puree.
 -d Juicy lamb chops served medium rare with a refreshing mint puree.`
-![addRecipe3.png](images/recipeCommands/addRecipe3.png)
+<br />Expected Output:<br />
+![addRecipe2.png](images/recipeCommands/addRecipe2.png)
```

Comment:
The output for steps no.2 is wrong (?) Typo right...hopefully?

Steps 1 also the space